cardano-node-chairman/test/Test/Cardano/Node/Chairman/ByronShelley.hs

```diff
@@ -652,6 +653,13 @@ prop_chairman = H.propertyOnce . H.workspace "chairman" $ \tempAbsPath -> do
         )
       )
 
+    return hProcess
+
+  -- Check for chairman success
+  forM_ (L.zip allNodes hChairmanProcesses) $ \(node, hProcess) -> do
```

Comment:
I think this is not right.  There should be a single `chairman` process connected to `allNodes`, not a `chairman` per node.

Reply:
Or is this going to be fixed in a subsequent PR?